Makefile

```diff
@@ -2125,18 +2128,9 @@ endif
 endif
 
 ifeq ($(ARCH), i386)
+ifeq ($(ARCH), x86_64)
```

Comment:
https://stackoverflow.com/questions/7656425/makefile-ifeq-logical-or?

Reply:
thanks! 
Changes reverted, I'm not sure about how implement the OR.